hwilib/commands.py

```diff
@@ -71,8 +72,8 @@ def enumerate(args):
         # Get keepkeys
         elif (d['vendor_id'], d['product_id']) in keepkey_device_ids:
             d_data['type'] = 'keepkey'
-        # Get ledgers
-        elif (d['vendor_id'], d['product_id']) in ledger_device_ids:
+        # Get ledgers, only take interface 0 to avoid HID keyboard
+        elif (d['vendor_id'], d['product_id']) in ledger_device_ids and d['interface_number'] == 0:
```

Comment:
btchip-python also checks for `usage_page` and checks that both `interface_number` and `usage_page` exist before getting their values. I think we should do that as well.

Reply:
I'm not sure what `user_page` is for, as I've only seen it as 0.

Reply:
fixed for interface number

Reply:
I found [this comment](https://github.com/digitalbitbox/mcu/blob/master/py/dbb_utils.py#L73) while looking through some Digital BitBox stuff:

```
# hidapi is not consistent across platforms
# usage_page works on Windows/Mac; interface_number works on Linux
```

So I think we still need the `usage_page` check

Reply:
ah, fixed!